frontend/src/pages/org/collection-editor.ts

```diff
@@ -1055,7 +1055,7 @@ export class CollectionEditor extends LiteElement {
         class="flex flex-col items-center justify-center text-center p-4 my-12"
       >
         <p class="text-neutral-400 text-center max-w-[24em]">
-          ${msg("Your organization doesn't have any uploaded Archive Data.")}
+          ${msg("Your organization hasn't uploaded any archived items.")}
```

Comment:
```suggestion
          ${msg("Your organization doesn't have any uploads.")}
```

Or something like 'No uploads found'. I think if we can mention the specific type of object that's referred here.